test/TEST_RESULTS.md

```diff
@@ -0,0 +1,18 @@
+
+### Method: heat
+
+Method-heat tests serve to test basic functionality of the "heat" method in Enzo-E.
+
+#### HEAT (serial)
+
+000000 |  000200 | 000400 
+---|---|---
+![](test/MethodHeat/Heat-1/method_heat-temp-1-000000.png) | ![](test/MethodHeat/Heat-1/method_heat-temp-1-000200.png) | ![](test/MethodHeat/Heat-1/method_heat-temp-1-000400.png)
+![](test/MethodHeat/Heat-1/method_heat-mesh-1-000000.png) | ![](test/MethodHeat/Heat-1/method_heat-mesh-1-000200.png) | ![](test/MethodHeat/Heat-1/method_heat-mesh-1-000400.png)
+
+#### HEAT (parallel)
+
+000000 |  000200 | 000400 
+---|---|---
+![](test/MethodHeat/Heat-8/method_heat-temp-8-000000.png) | ![](test/MethodHeat/Heat-8/method_heat-temp-8-000200.png) | ![](test/MethodHeat/Heat-8/method_heat-temp-8-000400.png)
+![](test/MethodHeat/Heat-8/method_heat-mesh-8-000000.png) | ![](test/MethodHeat/Heat-8/method_heat-mesh-8-000200.png) | ![](test/MethodHeat/Heat-8/method_heat-mesh-8-000400.png)
```

Comment:
What is the purpose of this file? Why does it only mention the Method-heat tests?